src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -184,6 +187,7 @@ private static HttpContext Copy(HttpContext source)
             target.Request.RouteValues = source.Request.RouteValues;
             target.Connection.RemoteIpAddress = source.Connection.RemoteIpAddress;
             target.RequestServices = source.RequestServices;
+            target.User = source.User;
```

Comment:
This could fix User context

Reply:
Ahah! I am laughing... 🤣 
You've made my day!

Reply:
So, do you approve or not?